lxd/db/network_zones.go

```diff
@@ -147,7 +148,7 @@ func (c *Cluster) GetNetworkZone(name string) (int64, string, *api.NetworkZone,
 	})
```

Comment:
@stgraber whilst doing this work I noticed that there is `GetNetworkZone(name string)` and `GetNetworkZoneByProject(projectName string, name string)`. Can both of these functions be correct? If a zone can exist in a project, it suggests one can have the same zone in another project, which would then make `GetNetworkZone(name)` incorrect?

Reply:
@tomponline a given zone can only ever exist in a single project.

`GetNetworkZoneByProject` is used to retrieve it for a user of the given project whereas `GetNetworkZone` is used to check if a zone already exists across all projects (used at creation time).

Reply:
I see, thanks for the clarification. Looking at the source code now its also used at DNS server load time.